### PR TITLE
Upgrade NuGetRestoreV1 to NodeJS16

### PR DESCRIPTION
**Pull request by sergey-koryshev**
**Task name**: NuGetRestoreV1

**Description**: This PR upgrades NuGetRestoreV1 to NodeJS 16

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by Roman-Shchukin — APPROVED** [review]

**Review by ismayilov-ismayil — APPROVED** [review]

File: Tasks/NuGetRestoreV1/package-lock.json
Comment (vmapetr) [inline Tasks/NuGetRestoreV1/package-lock.json:4]: We should probably stick to version 1.

Comment (sergey-koryshev) [reply Tasks/NuGetRestoreV1/package-lock.json]: fixed

**Comment by sergey-koryshev** [comment]
@microsoft/azure-artifacts-packages, could you please take a look and approve the PR? I will appreciate it if you will check that everything work as expected on your side

**Comment by JohnSchmeichel** [comment]
> @microsoft/azure-artifacts-packages, could you please take a look and approve the PR? I will appreciate it if you will check that everything work as expected on your side

You've checked the "Checked that applied changes work as expected" checkbox, can you please describe what testing you've done and what you expect from the Artifacts team here?

**Comment by sergey-koryshev** [comment]
Hi @JohnSchmeichel. Thank you for response. Since Artifacts team is codeowner of this task I asked to review the PR and test the changes. From my side I ran some dummy pipeline with updated task and also ran tests locally.

**Pull request closed without merging**
